sdk-core/src/main/java/care/data4life/sdk/migration/CompatibilityService.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.migration
+
+import care.data4life.sdk.ApiService
+import care.data4life.sdk.network.model.NetworkModelContract.EncryptedRecord
+import care.data4life.sdk.tag.Annotations
+import care.data4life.sdk.tag.TaggingContract
+import care.data4life.sdk.tag.Tags
+import io.reactivex.Observable
+import io.reactivex.Single
+import io.reactivex.functions.BiFunction
+
+class CompatibilityService internal constructor(
```

Comment:
I think this name is a bit too generic. Compatibility of what? is it going to be used for as a catch all for all legacy compatibility issues? Maybe would be nice if it was more specific, maybe adding the word Record somewhere

Reply:
Agreed

Reply:
done